specification/data-semantic-conventions.md

```diff
@@ -249,70 +279,118 @@ but due to `http.scheme`, `http.host` and `http.target` being set, it would be r
 As explained above, these separate values are preferred but if for some reason the URL is available but the other values are not,
 URL can replace `http.scheme`, `http.host` and `http.target`.
 
-## Databases client calls
+## Databases (SQL and NoSQL) client calls
+
+> 🚧 WORK IN PROGRESS
 
 For database client call the `SpanKind` MUST be `Client`.
 
 Span `name` should be set to low cardinality value representing the statement
-executed on the database. It may be stored procedure name (without argument), sql
+executed on the database. It may be stored procedure name (without argument), SQL
 statement without variable arguments, etc. When it's impossible to get any
 meaningful representation of the span `name`, it can be populated using the same
 value as `db.instance`.
```

Comment:
`db.instance` is used here but it has been removed from table below. I think we should replace it also here or add some description below what `db.instance` should contain.